static-assets/components/cstudio-dialogs/new-content-type.js

```diff
@@ -164,7 +175,10 @@ CStudioAuthoring.Dialogs.NewContentType = CStudioAuthoring.Dialogs.NewContentTyp
       objectTypeEl: document.getElementById('contentTypeObjectType')
     };
 
+    // 'Type' dropdown
     YEvent.on('contentTypeObjectType', 'change', function () {
+      self.onSetDirty(true);
```

Comment:
Should avoid setting this every keystroke. Set only when it changes from blank to not-blank and backwards and/or make sure react-side is also not setting state on every keystroke.